services/agents/mongodb_test.go

```diff
@@ -49,10 +49,8 @@ func TestMongodbExporterConfig225(t *testing.T) {
 		TemplateLeftDelim:  "{{",
 		TemplateRightDelim: "}}",
 		Args: []string{
-			// temporarily disabled by PMM-9304 until we have more information about the resources
-			// consumption for these 2 collectors.
-			// "--collector.dbstats",
-			// "--collector.topmetrics",
+			"--collector.diagnosticdata",
+			"--collector.replicasetstatus",
```

Comment:
if I got it right these two flags were added recently, so we should handle version.
different flags should be used for 2.25 and 2.26